libs/api/domains/disability-license/src/lib/disability-license.resolver.ts

```diff
@@ -0,0 +1,25 @@
+import { Args, Directive, Mutation, Query, Resolver } from '@nestjs/graphql'
+
+import { Scope, UseGuards } from '@nestjs/common'
+
+import {
+  AuthMiddleware,
+  BypassAuth,
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import type { User } from '@island.is/auth-nest-tools'
+import { DisabilityLicenseService } from '@island.is/clients/disability-license'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver()
+export class DisabilityLicenseResolver {
+  constructor(private disabilityLicenseApi: DisabilityLicenseService) {}
+
+  @Query(() => Boolean)
```

Comment:
This should have some Island.is scope. Maybe start with "ApiScope.internal" so only the authenticated user can access this?

Reply:
Github isn't showing it here but added ApiScope.internal at the resolver level.